components/d2l-activity-editor/d2l-activity-assignment-editor/d2l-activity-assignment-editor-detail.js

```
@@ -74,9 +75,18 @@ class AssignmentEditorDetail extends ErrorHandlingMixin(SirenFetchMixinLit(Entit
 
 	_saveName(value) {
 		if (super._entity.canEditName()) {
+			const saveStatus = this.shadowRoot.querySelector('#save-status');
+			saveStatus.start();
+
 			const action = super._entity.getSaveNameAction();
 			const fields = [{ 'name': 'name', 'value': value }];
-			this._performSirenAction(action, fields);
+			this._performSirenAction(action, fields)
+				.then(() => {
+					saveStatus.end();
+				})
+				.catch(() => {
+					saveStatus.error();
```

We should probably do something nicer here, although unclear what - the `d2l-save-status` component doesn't appear to support showing an error or anything, so we probably would need to surface the error somewhere else.